<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>QuickFace</class>
 <widget class="QWidget" name="QuickFace">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>900</width>
    <height>1080</height>
   </rect>
  </property>
  <property name="minimumSize">
   <size>
    <width>900</width>
    <height>1080</height>
   </size>
  </property>
  <property name="maximumSize">
   <size>
    <width>900</width>
    <height>1080</height>
   </size>
  </property>
  <property name="windowTitle">
   <string>QuickFace</string>
  </property>
  <property name="windowIcon">
   <iconset resource="qml.qrc">
    <normaloff>:/image/image/QuickFace.png</normaloff>:/image/image/QuickFace.png</iconset>
  </property>
  <property name="styleSheet">
   <string notr="true">background-color: rgb(235, 235, 235);
</string>
  </property>
  <widget class="QWidget" name="verticalLayoutWidget_2">
   <property name="geometry">
    <rect>
     <x>680</x>
     <y>150</y>
     <width>202</width>
     <height>911</height>
    </rect>
   </property>
   <layout class="QVBoxLayout" name="verticalLayout_2">
    <property name="spacing">
     <number>30</number>
    </property>
    <property name="sizeConstraint">
     <enum>QLayout::SetNoConstraint</enum>
    </property>
    <item>
     <widget class="QPushButton" name="StartCam">
      <property name="minimumSize">
       <size>
        <width>100</width>
        <height>102</height>
       </size>
      </property>
      <property name="maximumSize">
       <size>
        <width>200</width>
        <height>203</height>
       </size>
      </property>
      <property name="autoFillBackground">
       <bool>false</bool>
      </property>
      <property name="styleSheet">
       <string notr="true"/>
      </property>
      <property name="text">
       <string/>
      </property>
      <property name="icon">
       <iconset resource="qml.qrc">
        <normaloff>:/image/image/Start.png</normaloff>:/image/image/Start.png</iconset>
      </property>
      <property name="iconSize">
       <size>
        <width>197</width>
        <height>200</height>
       </size>
      </property>
      <property name="checkable">
       <bool>false</bool>
      </property>
      <property name="checked">
       <bool>false</bool>
      </property>
      <property name="autoDefault">
       <bool>false</bool>
      </property>
      <property name="default">
       <bool>false</bool>
      </property>
      <property name="flat">
       <bool>false</bool>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QPushButton" name="SaveObj">
      <property name="minimumSize">
       <size>
        <width>200</width>
        <height>203</height>
       </size>
      </property>
      <property name="maximumSize">
       <size>
        <width>200</width>
        <height>203</height>
       </size>
      </property>
      <property name="autoFillBackground">
       <bool>false</bool>
      </property>
      <property name="styleSheet">
       <string notr="true"/>
      </property>
      <property name="text">
       <string/>
      </property>
      <property name="icon">
       <iconset resource="qml.qrc">
        <normaloff>:/image/image/Save.png</normaloff>:/image/image/Save.png</iconset>
      </property>
      <property name="iconSize">
       <size>
        <width>198</width>
        <height>201</height>
       </size>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QPushButton" name="Mode">
      <property name="minimumSize">
       <size>
        <width>200</width>
        <height>203</height>
       </size>
      </property>
      <property name="maximumSize">
       <size>
        <width>200</width>
        <height>203</height>
       </size>
      </property>
      <property name="autoFillBackground">
       <bool>false</bool>
      </property>
      <property name="styleSheet">
       <string notr="true"/>
      </property>
      <property name="text">
       <string/>
      </property>
      <property name="icon">
       <iconset resource="qml.qrc">
        <normaloff>:/image/image/Mode.png</normaloff>:/image/image/Mode.png</iconset>
      </property>
      <property name="iconSize">
       <size>
        <width>198</width>
        <height>201</height>
       </size>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QPushButton" name="Quit">
      <property name="minimumSize">
       <size>
        <width>200</width>
        <height>203</height>
       </size>
      </property>
      <property name="maximumSize">
       <size>
        <width>200</width>
        <height>203</height>
       </size>
      </property>
      <property name="autoFillBackground">
       <bool>false</bool>
      </property>
      <property name="styleSheet">
       <string notr="true"/>
      </property>
      <property name="text">
       <string/>
      </property>
      <property name="icon">
       <iconset resource="qml.qrc">
        <normaloff>:/image/image/Quit.png</normaloff>
        <normalon>:/image/image/QuitOn.png</normalon>:/image/image/Quit.png</iconset>
      </property>
      <property name="iconSize">
       <size>
        <width>198</width>
        <height>201</height>
       </size>
      </property>
     </widget>
    </item>
   </layout>
  </widget>
  <widget class="QFrame" name="frame_3">
   <property name="geometry">
    <rect>
     <x>680</x>
     <y>20</y>
     <width>201</width>
     <height>111</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">background-image: url(:/image/image/Logo.png);</string>
   </property>
   <property name="frameShape">
    <enum>QFrame::StyledPanel</enum>
   </property>
   <property name="frameShadow">
    <enum>QFrame::Raised</enum>
   </property>
  </widget>
  <widget class="QWidget" name="verticalLayoutWidget">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>20</y>
     <width>644</width>
     <height>1046</height>
    </rect>
   </property>
   <layout class="QVBoxLayout" name="verticalLayout">
    <property name="spacing">
     <number>20</number>
    </property>
    <item>
     <layout class="QVBoxLayout" name="verticalLayout_5">
      <property name="spacing">
       <number>0</number>
      </property>
      <item>
       <widget class="QFrame" name="frame_4">
        <property name="minimumSize">
         <size>
          <width>640</width>
          <height>30</height>
         </size>
        </property>
        <property name="maximumSize">
         <size>
          <width>640</width>
          <height>30</height>
         </size>
        </property>
        <property name="styleSheet">
         <string notr="true">background-color: rgb(179, 179, 179);</string>
        </property>
        <property name="frameShape">
         <enum>QFrame::StyledPanel</enum>
        </property>
        <property name="frameShadow">
         <enum>QFrame::Raised</enum>
        </property>
        <widget class="QLabel" name="label">
         <property name="geometry">
          <rect>
           <x>0</x>
           <y>0</y>
           <width>191</width>
           <height>30</height>
          </rect>
         </property>
         <property name="font">
          <font>
           <family>Adobe Garamond Pro Bold</family>
           <pointsize>16</pointsize>
          </font>
         </property>
         <property name="text">
          <string>Capture Video</string>
         </property>
         <property name="alignment">
          <set>Qt::AlignCenter</set>
         </property>
        </widget>
       </widget>
      </item>
      <item>
       <widget class="QFrame" name="frame_2">
        <property name="minimumSize">
         <size>
          <width>640</width>
          <height>480</height>
         </size>
        </property>
        <property name="maximumSize">
         <size>
          <width>640</width>
          <height>480</height>
         </size>
        </property>
        <property name="styleSheet">
         <string notr="true">background-color: rgb(0, 0, 0);</string>
        </property>
        <property name="frameShape">
         <enum>QFrame::Panel</enum>
        </property>
        <property name="frameShadow">
         <enum>QFrame::Plain</enum>
        </property>
        <property name="lineWidth">
         <number>2</number>
        </property>
        <widget class="QLabel" name="CapturePic">
         <property name="geometry">
          <rect>
           <x>0</x>
           <y>0</y>
           <width>640</width>
           <height>480</height>
          </rect>
         </property>
         <property name="minimumSize">
          <size>
           <width>640</width>
           <height>480</height>
          </size>
         </property>
         <property name="text">
          <string/>
         </property>
        </widget>
       </widget>
      </item>
     </layout>
    </item>
    <item>
     <layout class="QVBoxLayout" name="verticalLayout_6">
      <property name="spacing">
       <number>0</number>
      </property>
      <item>
       <widget class="QFrame" name="frame_5">
        <property name="minimumSize">
         <size>
          <width>640</width>
          <height>30</height>
         </size>
        </property>
        <property name="maximumSize">
         <size>
          <width>640</width>
          <height>30</height>
         </size>
        </property>
        <property name="styleSheet">
         <string notr="true">background-color: rgb(179, 179, 179);</string>
        </property>
        <property name="frameShape">
         <enum>QFrame::StyledPanel</enum>
        </property>
        <property name="frameShadow">
         <enum>QFrame::Raised</enum>
        </property>
        <widget class="QLabel" name="label_2">
         <property name="geometry">
          <rect>
           <x>0</x>
           <y>0</y>
           <width>281</width>
           <height>30</height>
          </rect>
         </property>
         <property name="font">
          <font>
           <family>Adobe Garamond Pro Bold</family>
           <pointsize>16</pointsize>
          </font>
         </property>
         <property name="text">
          <string>Reconstruction Model</string>
         </property>
         <property name="alignment">
          <set>Qt::AlignCenter</set>
         </property>
        </widget>
       </widget>
      </item>
      <item>
       <widget class="QFrame" name="frame">
        <property name="minimumSize">
         <size>
          <width>640</width>
          <height>480</height>
         </size>
        </property>
        <property name="maximumSize">
         <size>
          <width>640</width>
          <height>480</height>
         </size>
        </property>
        <property name="styleSheet">
         <string notr="true">background-color: rgb(0, 0, 0);</string>
        </property>
        <property name="frameShape">
         <enum>QFrame::WinPanel</enum>
        </property>
        <property name="frameShadow">
         <enum>QFrame::Plain</enum>
        </property>
        <property name="lineWidth">
         <number>2</number>
        </property>
        <property name="midLineWidth">
         <number>0</number>
        </property>
        <widget class="QLabel" name="ModelPic">
         <property name="geometry">
          <rect>
           <x>0</x>
           <y>0</y>
           <width>640</width>
           <height>480</height>
          </rect>
         </property>
         <property name="minimumSize">
          <size>
           <width>640</width>
           <height>480</height>
          </size>
         </property>
         <property name="text">
          <string/>
         </property>
        </widget>
       </widget>
      </item>
     </layout>
    </item>
   </layout>
  </widget>
 </widget>
 <resources>
  <include location="qml.qrc"/>
 </resources>
 <connections/>
</ui>
